themes/edx.org/lms/templates/course_modes/choose.html

```diff
@@ -73,7 +73,13 @@ <h3 class="title">${_("Sorry, there was an error when trying to enroll you")}</h
                 <article class="register-choose content-main">
                     <header class="page-header content-main">
                         <h3 class="title">
+                        % if show_enterprise_context:
```

Comment:
I would like to see all of this decision logic and strings moved into the view, and just a simple context parameter referenced here in the template.